samples/Zirku/Zirku.Api1/Program.cs

```diff
@@ -28,11 +28,13 @@
         options.UseAspNetCore();
     });
 
+builder.Services.AddCors();
 builder.Services.AddAuthentication(OpenIddictValidationAspNetCoreDefaults.AuthenticationScheme);
 builder.Services.AddAuthorization();
 
 var app = builder.Build();
 
+app.UseCors(b => b.AllowAnyHeader().AllowAnyMethod().AllowAnyOrigin());
```

Comment:
`AllowAnyOrigin()` is convenient, but not something we should recommend in samples. Can you please just add the origin of the SPA app?

Reply:
Looks like it's the only remaining thing to address before merging this PR 😃

Reply:
Aah missed that one. I've edited directly from Github, I think it should be fine.